CHANGELOG.md

```diff
@@ -21,7 +21,9 @@ and this project adheres to Rust's notion of
   - `impl memuse::DynamicUsage for Nullifier`
 - `orchard::note_encryption`:
   - `impl memuse::DynamicUsage for OrchardDomain`
-- `orchard::builder::SpendInfo::new`
+- `orchard::builder`:
+  - `{SpendInfo::new, InputView, OutputView}`
+  - `Builder::{spends, outputs}`
```

Comment:
This should be moved to the `Unreleased` section.

Reply:
Fixed!